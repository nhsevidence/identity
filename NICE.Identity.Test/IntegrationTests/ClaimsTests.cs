using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Threading.Tasks;
using NICE.Identity.Authorisation.WebAPI.Models;
using NICE.Identity.Test.Infrastructure;
using Xunit;

namespace NICE.Identity.Test.IntegrationTests
{
	public class ClaimsTests : TestBase
	{
		[Fact]
		public async Task GetClaims()
		{
			//Arrange
			var context = GetContext();
			TestData.AddAll(ref context);
			var client = GetClient(context);

			//Act
			var response = await client.GetAsync("api/claims/1");
			response.EnsureSuccessStatusCode();
			var responseString = await response.Content.ReadAsStringAsync();

			//Assert
			responseString.ShouldMatchApproved();
		}

		
	}
}
