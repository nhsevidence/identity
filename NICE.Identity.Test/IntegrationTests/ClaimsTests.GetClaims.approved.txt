[
    {
        "roleId":1,
        "roleName":"Admin"
    }
]